examples/i2c/client_pn532.c

```diff
@@ -148,11 +165,12 @@ void client_main(void) {
             LOG_CLIENT_ERR("Timed out waiting for a card\n");
         }
 
-        delay_ms(1000);
+        delay_ms(500);
```

Comment:
Any reason for this change? Just curious, I don't see anything wrong with it if it all still works though.

Reply:
no reason , I think I just wanted output to be faster